web/viewer.html

```diff
@@ -211,15 +217,15 @@
                   <span data-l10n-id="toggle_sidebar_label">Toggle Sidebar</span>
                 </button>
                 <div class="toolbarButtonSpacer"></div>
-                <button id="viewFind" class="toolbarButton group hiddenSmallView" title="Find in Document" tabindex="12" data-l10n-id="findbar">
+                <button id="viewFind" class="toolbarButton group" title="Find in Document" tabindex="12" data-l10n-id="findbar">
                    <span data-l10n-id="findbar_label">Find</span>
                 </button>
                 <div class="splitToolbarButton">
-                  <button class="toolbarButton pageUp" title="Previous Page" id="previous" tabindex="13" data-l10n-id="previous">
+                  <button class="toolbarButton pageUp hiddenSmallView" title="Previous Page" id="previous" tabindex="13" data-l10n-id="previous">
```

Comment:
Please add `hiddenSmallView` to the `<div class="splitToolbarButton">` just above instead, since adding it to each element seems unnecessary (and more prone to errors if you forget one child element).